pkgs/by-name/di/direwolf-unstable/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  direwolf,
+  espeak,
+  nix-update-script,
+  hamlibSupport ? true,
+  gpsdSupport ? true,
+  extraScripts ? false,
+}:
+
+(direwolf.override {
+  inherit hamlibSupport gpsdSupport extraScripts;
+}).overrideAttrs
+  (finalAttrs: {
```

```suggestion
  (oldAttrs: {
```
The function input for `overrideAttrs` is the old derivation before the overrides. `finalAttrs` is the derivation after it's been overridden.